app/assets/stylesheets/components/buttons.scss

```diff
@@ -124,6 +124,9 @@
   // See: https://css-tricks.com/slightly-careful-sub-elements-clickable-things/
   & > * {
     pointer-events: none;
+    + input {
```

Comment:
I've tested this locally and this works. We needed to do this because there is an input of type file in a button for image uploads on the create post page. We should rework that in another PR.